utils/prescription_id_generator.py

```diff
@@ -0,0 +1,40 @@
+import uuid
```

Comment:
If this file is for manual use, maybe a mention in the README.

Is generating a short-form from a short-form correct?

Also, if the ODS code is fixed, consider making it a constant alongside CHECK_DIGIT_VALUES, as it doesn't need to be an argument.

Reply:
Yes. Correct

Reply:
OK, cheers.